frontend/src/utils/forms/index.js

```diff
@@ -19,6 +20,12 @@ const widgets = {
   URLWidget: URLWidget,
   AltDateWidget: AltDateWidget,
   DateWidget: DateWidget,
+  EmailWidget: EmailWidget,
+};
+
+export const customFormats = {
+  url: /[-a-zA-Z0-9@:%_\+.~#?&//=]{2,256}\.[a-z]{2,4}\b(\/[-a-zA-Z0-9@:%_\+.~#?&//=]*)?/,
+  email: /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/,
```

Comment:
Did we came with this regex ourselves? 🤔 
If not, let's double check the license of the code snippet (attribution if required)